eng/scripts/Pack-Nuget.ps1

```diff
@@ -75,6 +75,7 @@ try {
             -replace '\$\(PackageDescription\)', $serverProjectProperties.Description `
```

Comment:
If you use string.Replace here, you don't have to escape anything

```suggestion
            .Replace('$(PackageDescription)', $serverProjectProperties.Description) `
```

Also, the right side of a `-replace` is still interpreted as a regex replacement pattern   `$1something$2`.  To be safe, you should call [Regex]::Replace() on each of the right hand side args unless you know they don't contain and regex replacement special characters